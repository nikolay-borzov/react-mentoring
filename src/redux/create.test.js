import configureStore from './create'

describe('store', () => {
  it('creates store with initial state', () => {
    const { store } = configureStore()

    expect(store.getState()).toMatchSnapshot()
  })
})
