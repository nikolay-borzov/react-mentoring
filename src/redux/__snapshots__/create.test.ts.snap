// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store creates store with initial state 1`] = `
Object {
  "search": Object {
    "films": Object {
      "isFetching": false,
      "items": Array [],
      "limit": 0,
      "total": 0,
    },
    "searchParams": Object {
      "filter": Array [],
      "limit": 15,
      "search": "",
      "searchBy": "title",
      "sortBy": "release_date",
      "sortOrder": "desc",
    },
  },
  "view": Object {
    "film": Object {
      "isFetching": false,
    },
    "relatedFilms": Object {
      "films": Object {
        "isFetching": false,
        "items": Array [],
        "limit": 0,
        "total": 0,
      },
      "searchParams": Object {
        "filter": Array [],
        "limit": 20,
        "search": "",
        "searchBy": "genres",
        "sortBy": "vote_average",
        "sortOrder": "desc",
      },
    },
  },
}
`;
